packages/css/src/components/whitebox/whitebox.template.ts

```diff
@@ -0,0 +1,44 @@
+import { Whitebox } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map';
+
+import { iconTemplate } from '../icon/icon.template';
+import { imageTemplate } from '../image/image.template';
+
+export function whiteboxTemplate({ count, icon, iconText, image, label, richContent, title }: Whitebox<TemplateResult>) {
+  return html`
+    <div class="dso-whitebox ${classMap({ 'dso-has-counter': !!(count || (icon && iconText)) })}">
+      ${icon && iconText
+        ? html`
+          <div class="dso-step-counter">
+            ${iconTemplate({ icon: 'check' })}
```

Comment:
Dat hier `check` staat is ook maar 'n voorbeeld.. Volgens mij kan hier elk icoon gebruikt worden.

Reply:
Op regel 11 komt ie zelfs binnen...